pkg/services/baremetal/host/host.go

```diff
@@ -1426,9 +1426,13 @@ func (s *Service) actionDeprovisioning() actionResult {
 			IP:         s.scope.HetznerBareMetalHost.Spec.Status.GetIPAddress(),
 		})
 		out := sshClient.ResetKubeadm()
-		if err := handleSSHError(out); err != nil {
+		s.scope.Info("Output of ResetKubeadm", "stdout", out.StdOut, "stderr", out.StdErr, "err", out.Err)
```

Comment:
you only need this for debugging - not in the controller I guess

Reply:
@janiskemper I would like to keep that. I hope it is ok for you, since it does not happen often.